docs/overview.rst

```diff
@@ -56,52 +58,50 @@ administrators, the sources, and the journalists.
 Servers
 ~~~~~~~
 
-At SecureDrop's heart is a pair of severs: the *Application (“App”)
-Server*, which runs the core SecureDrop software, and the *Monitor
-(“Mon”) Server*, which keeps track of the *Application Server* and sends
-out alerts if there's a problem. These two servers run on dedicated
-hardware connected to a dedicated firewall appliance. They are typically
-located physically inside the newsroom.
+At SecureDrop's heart is a pair of severs: the *Application (“App”) Server*, 
+which runs the core SecureDrop software, and the *Monitor (“Mon”) Server*, 
+which keeps track of the *Application Server* and sends out alerts if there's a 
+problem. These two servers run on dedicated hardware connected to a dedicated 
+firewall appliance. They are typically located physically inside the newsroom.
 
 Administrators
 ~~~~~~~~~~~~~~
 
-The SecureDrop servers are managed by a systems administrator; for
-larger newsrooms, there may be a team of systems administrators. The
-administrator uses a dedicated *Admin Workstation* running
-`Tails <https://tails.boum.org>`__ and connects to the App and Mon
-servers over authenticated `Tor Hidden
-Services <https://www.torproject.org/docs/hidden-services.html>`__ and
-manages them using `Ansible <http://www.ansible.com/>`__.
+The SecureDrop servers are managed by a systems administrator; for larger 
+newsrooms, there may be a team of systems administrators. The administrator 
+uses a dedicated *Admin Workstation* running `Tails <https://tails.boum.org>`__ 
+and connects to the App and Mon servers over authenticated `Tor Hidden Services 
+<https://www.torproject.org/docs/hidden-services.html>`__ and manages them 
+using `Ansible <http://www.ansible.com/>`__.
```

Comment:
Ah, I see. That's annoying. Just leave it alone for now, if we do that I agree it should be done all at once. If it ain't broke, don't fix it.